cmake/kokkos_compiler_id.cmake

```diff
@@ -64,7 +64,7 @@ ELSEIF(KOKKOS_CXX_COMPILER_ID STREQUAL Intel)
     MESSAGE(FATAL_ERROR "${KOKKOS_MESSAGE_TEXT}")
   ENDIF()
 ELSEIF(KOKKOS_CXX_COMPILER_ID STREQUAL NVIDIA)
-  IF(KOKKOS_CXX_COMPILER_VERSION VERSION_LESS 7.0.28)
+  IF(KOKKOS_CXX_COMPILER_VERSION VERSION_LESS 9.0.69)
```

Comment:
I don't know about making this a fatal error.  I would raise a warning.

Reply:
No this is ok as a fatal error. We will drop the code path for pre-CUDA 9 shuffle operations. I.e. we will only have the shfl_sync versions in the code which didn't exist in CUDA 8.0.